it/uk/gov/hmrc/agentsubscriptionfrontend/controllers/SessionDataMissingSpec.scala

```diff
@@ -26,6 +26,6 @@ trait SessionDataMissingSpec {
 
   def resultShouldBeSessionDataMissing(result: Result)(implicit timeout: Timeout): Unit = {
```

Comment:
Could maybe rename this method to something like `resultShouldRedirectToCheckBusinessType`.
Also, it's called in 3 different tests, but the names of those tests still mention the Check Agency Status page, so they need a quick rename too.

Reply:
Actually, the original name `resultShouldBeSessionDataMissing` is self-documenting. Perhaps it should stay with that name.